app/Http/Controllers/Product/ProductController.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace App\Http\Controllers\Product;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Product;
+use App\Brand;
+use App\Category;
+
+class ProductController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $products = Product::all()->where('seller_id',auth()->id());
+        return view('seller.product.index_product')->with('products',$products);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $brands = Brand::get_available_brands();
+        $categories = Category::all();
+        return view('seller.product.create_product')->with('brands',$brands)->with('categories',$categories);
```

Comment:
you can use the variables within a single associative array instead of multiple with() method calls chained one after the other. 

change $brands to $data["brands"] and change $categories to $data["categories"] and pass the variable $data into the view() function as second parameter to remove all with() methods or at least use only one with() method call. This would enhance code readability.